### PR TITLE
Admin can view catch return results - validate input

### PR DESCRIPTION
**Pull request by nabeelamir-defra**
https://eaflood.atlassian.net/browse/IWTF-1860

validate admin input

### CODE REVIEW

File: src/test/java/uk/gov/defra/datareturns/test/licence/LicenceLookupIT.java [hunk 1]
```diff
@@ -54,11 +54,16 @@ public void testLicenceReadOnly() {
...
 
     @Test
-    public void testLFullicenceLookupB7A111() {
+    public void testLFullLicenceLookupB7A111() {
         getEntity("/licence/full/00081019-1WS3JP4-B7A718")
                 .statusCode(HttpStatus.OK.value());
     }
 
+    @Test
+    public void testFullLicenceLookupInvalid() {
```
Comment (jaucourt) [inline src/test/java/uk/gov/defra/datareturns/test/licence/LicenceLookupIT.java:63]: I think it's probably worth having a few invalid test cases (similarly, above, have a few valid options too). Improves the use-case examples and possibly covers some edge cases too

**Review by jaucourt — CHANGES_REQUESTED** [review]

**Review by jaucourt — APPROVED** [review]
